.env

```diff
@@ -1,4 +1,4 @@
 AUTH_PORT=3500
 SERVER_PORT=5001
-DB_URI=mongodb+srv://dudge1:YjJYaLx1QiyutHHR@mea-bank.zbjcduu.mongodb.net/bankDBIP3?retryWrites=true&w=majority&appName=MEA-Bank
-NODE_ENV=dev
\ No newline at end of file
+DB_URI=mongodb+srv://dudge1:YjJYaLx1QiyutHHR@mea-bank.a4ca1bz.mongodb.net/bankDBIP3?retryWrites=true&w=majority&appName=MEA-Bank
```

Comment:
has this another new link?

Reply:
It'll match the one in the BE. Should be the up to date one that you now have locally.